### DISCUSSION TITLE
linux: please support the creation of symlinks to smb directories

### DISCUSSION THREAD

**Discussion by StuckiD** [doublecmd/doublecmd#2007] [Suggestions]
please support the creation of symlinks to smb directories.
that would be great.
best regards, daniel

I use:
Double Commander
Version: 1.1.11 beta
Revision: 410
Commit: f1b449635
Build date: 2024/04/01
Lazarus: 3.0.0.3
FPC: 3.2.2
Platform: x86_64-Linux-gtk2
OS version: Linux Mint 22 Wilma

Widgetset library: GTK 2.24.33

**Comment by alexx2000** [comment]
I guess it is impossible.

**Reply by StuckiD** [reply]
Thanks for the quick reply. I have now found a solution via /run/user/1000/gvfs/smb-share\:server\=servername.local\,share\=sharename/
This way I can now create symlinks (see photo).
![Bildschirmfoto DC 2024-12-12 16-49-44](https://github.com/user-attachments/assets/36e1200d-8109-456f-ad91-d9f7092cb0bd)

**Reply by alexx2000** [reply]
Such method can work only with a mounted share.
